index.html

```diff
@@ -5,4 +5,39 @@
     <link rel="stylesheet" type="text/css" href="css/reset.css">
     <link rel="stylesheet" type="text/css" href="css/style.css">
   </head>
+  <body>
+    <div>
```

Comment:
Make sure to look through your code before you merge it in: this div is empty & not doing anything for you.